nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/state/flow/flow.effects.ts

```diff
@@ -1126,6 +1126,24 @@ export class FlowEffects {
         { dispatch: false }
     );
 
+    navigateToViewStatusHistoryForCurrentProcessGroup$ = createEffect(
+        () =>
+            this.actions$.pipe(
+                ofType(FlowActions.navigateToViewStatusHistoryForCurrentProcessGroup),
+                concatLatestFrom(() => this.store.select(selectCurrentProcessGroupId)),
+                tap(([, processGroupId]) => {
+                    this.router.navigate([
+                        '/process-groups',
+                        processGroupId,
+                        ComponentType.ProcessGroup,
+                        processGroupId,
+                        'history'
+                    ]);
```

Comment:
The assumption for this route is that the second id is a child component of the Process Group specified by first id. I think adding a route for `/process-groups/{id}/history` would be more correct for opening the status history for the current Process Group.